packaging/suse/patches/0_change_versions.gem.patch

```diff
@@ -19,15 +19,15 @@ index cd17fc91be2a..cd5d5580f36d 100644
  
  # In order to create the Gemfile.lock required for packaging
  # meaning that it should contain only the production packages
-@@ -69,6 +69,7 @@ unless packaging?
+@@ -70,6 +70,7 @@ unless packaging?
      gem "pry-rails"
      gem "git-review", require: false
      gem "rack-mini-profiler", require: false
 +    gem "brakeman", require: false
```

Comment:
this green refers to the 0_change_versions.gem.patch , sorry I didn't realize it. Looks good.